test/com/facebook/buck/java/intellij/ProjectIntegrationTest.java

```diff
@@ -443,19 +443,32 @@ public void testBuckProjectWithMultipleLibrariesInOneBuildFile() throws IOExcept
   }
 
   @Test
-  public void testNonexistentTarget() throws IOException {
+  public void testProjectWithColon() throws IOException {
     ProjectWorkspace workspace = TestDataHelper.createProjectWorkspaceForScenario(
         this,
         "project1",
         temporaryFolder);
     workspace.setUp();
 
-    expectedException.expect(HumanReadableException.class);
```

Comment:
It's okay; we don't have great docs on this and we had to stop using Travis because we'd just get OOMs in random places of the build/setup.  Normally I try to fix things up on my own, but I've been a bit swamped today.